vcr.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |spec|
   spec.summary       = %q{Record your test suite's HTTP interactions and replay them during future test runs for fast, deterministic, accurate tests.}
   spec.description   = spec.summary
   spec.homepage      = "https://relishapp.com/vcr/vcr/docs"
-  spec.license       = "MIT-Hippocratic-1.2"
+  spec.license       = "MIT-Hippocratic-2.0"
```

Comment:
Why do you call this the MIT-hippocratic-2.0?
This is officially called at https://firstdonoharm.dev/ "The Hippocratic License 2.0"
So a tag should be best be "hippocratic-2.0" rather than the possibly confusing "mit-hippocratic-2.0" ...
@CoralineAda would you concur?

Reply:
Note that If you still have MIT-licensed code too, then you should consider using an SPDX-like license expression syntax such as: `MIT AND hippocratic-2.0`

Reply:
Since RubyGems doesn't have a spec for this, I suppose any will do.